BookingsApi/BookingsApi.DAL/Commands/UpdateHearingParticipantsCommand.cs

```diff
@@ -57,10 +57,14 @@ public async Task Handle(UpdateHearingParticipantsCommand command)
 
             _context.Update(hearing);
             
+            foreach (var removedParticipantId in command.RemovedParticipantIds)
+                hearing.RemoveParticipant(removedParticipantId, false);
```

Comment:
To handle an edge case where a participant is removed and the same account added. I.e. the panel member is changed from a participant to the judge but there is only a judge. If we remove the participants first, which we must to avoid the domain error where the participant is already in the hearing, there would be one participant left which is less than two throwing a domain error. By not validating that there is the correct number of participants till the end it gets around this.